scripts/monitor/rano_monitor/tarball_browser.py

```diff
@@ -50,10 +50,12 @@ def set_vars(self, contents_path):
 
     def __get_subjects(self):
         subjects = os.listdir(self.contents_path)
+        subjects = [subject for subject in subjects if not subject.startswith(".")]
         subject_timepoint_list = []
         for subject in subjects:
             subject_path = os.path.join(self.contents_path, subject)
             timepoints = os.listdir(subject_path)
+            timepoints = [timepoint for timepoint in timepoints if not timepoint.startswith(".")]
             subject_timepoint_list += [(subject, tp) for tp in timepoints]
 
```

Comment:
So this is added to avoid the generated MacOS hidden files? And we are assuming no subject names/timepoint would start with a .?

Reply:
Correct! I could make it more specific if we want to handle hidden cases, but I think that would be a very specific edge case to encounter